src/cljdoc/render/layout.clj

```diff
@@ -78,11 +78,50 @@
    {:style {:top (str TOP-BAR-HEIGHT "px") :left "16rem"}} ; CSS HACK
    contents])
 
+(defn meta-info-icon []
+  [:img#meta-icon.ma3
+   {:src "/meta-info-icon.svg"
+    :alt "meta info icon"
+    :width "60px"
+    :style {:position "fixed"
+            :right 0
+            :bottom 0
+            :background "white"}}])
```

Comment:
You can use Tachyons classes for this too: `.fixed.right-0.bottom-0.bg-white`.

For `width` there's various `w-N` classes, probably best to just play around with those and find something that works. (Or maybe just set the size int the icon URL as described on https://icon.now.sh)